### ISSUE TITLE
Visual label/heading is not provided for the chart: A11y_.NET 9 template_.NET 9 template_Home_Labels or Instrunctions.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26802]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG3.3.2; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Labels or Instructions; #MAC;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:
1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till chart section.
4. Observe that visual label/heading is not provided for the chart.

### Actual Result:
Visual label/heading is not provided for the chart.

### Expected Result:
Visual label/heading should be provided for the Chart.

### User Impact:
Users will face difficulties in knowing the purpose of the Chart as visual label is not provided for the Chart.

### Attachment:
<img width="1021" alt="Image" src="https://github.com/user-attachments/assets/daab91e0-e239-48fe-8d70-fc32c944473e" />

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by sheiksyedm** [comment]
Duplicate #26232

**Comment by AnKushSingh05** [comment]
> Duplicate [#26232](https://github.com/dotnet/maui/issues/26232)

Hi @sheiksyedm, the bug you provided is for Win32, while the logged bug is for the MAC platform, so they are not duplicates.

**Comment by tj-devel709** [comment]
fix is in this PR: https://github.com/dotnet/maui-samples/pull/569

**Comment by AnKushSingh05** [comment]
Issue is fixed

<img width="1031" alt="Image" src="https://github.com/user-attachments/assets/62b23d23-c7b4-4475-8022-1f8bb40c5e90" />
